### ISSUE TITLE
Support devcontainer feature build/test/publish.

### ISSUE THREAD

**Issue by Clockwork-Muse** [devcontainers/ci#224]
A large part of the ecosystem of devcontainers is the use of features to extend the environment, and potentially the IDE.  Currently the scripts here do not support building, testing, or publishing devcontainer features, meaning either the feature must be baked into the base image (hindering reusability), or creating custom scripts to directly call the cli.  This organization does have an additional repository that contains a GitHub action that can handle features, but unfortunately doesn't support azure at all.

**Comment by stuartleeks** [comment]
@Clockwork-Muse - if I've understood correctly, you're looking for something like the functionality of https://github.com/devcontainers/action but for Azure DevOps?

**Comment by Clockwork-Muse** [comment]
Yeah... but this repo doesn't support this feature for GitHub actions either...

**Comment by stuartleeks** [comment]
This repo (devcontainers/ci) is focussed on running build steps inside a dev container, not on managing features/templates. 
The devcontainers/action repo is targetted at packaging dev container features and templates (but only has a GitHub action)

I wanted to make sure I understood which functionality you are looking for. It sounds like it's the feature testing/publishing flow that you are interested in (but for Azure DevOps)?

**Comment by Clockwork-Muse** [comment]
Yes, primarily.

**Comment by mmartinortiz** [comment]
Hi, this is an old issue, but I could not find any functionality for testing/publishing features or templates in Azure DevOps. Are there plans for it?
